database/migrations/model/Version20220629180748.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Database\Migrations\Model;
+
+use Doctrine\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema as Schema;
+use LaravelDoctrine\Migrations\Schema\Builder;
+use LaravelDoctrine\Migrations\Schema\Table;
+
+class Version20220629180748 extends AbstractMigration
+{
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema): void
+    {
+        $sql = <<<SQL
```

Comment:
use the schema builder so u could ask if exists or no the column before create it
update to enum on a later migration

Reply:
Done